csrc/device_lower/pass/index.cpp

```diff
@@ -1282,6 +1282,34 @@ void IndexLowering::handleGroupedGridWelford(
   }
 }
 
+namespace {
+
+Val* getMBarrierSmemAddr(TensorView* mbarrier) {
```

Comment:
Not sure if it worth to make `getMBarrierSmemAddr` a util function, so the test can use it instead of rewrite.

Reply:
Added as `lower_utils::u32IndexScalarSmemTv`